OIOUBL/src/Management/COD13646.Management.al

```diff
@@ -55,4 +55,19 @@ codeunit 13646 "OIOUBL-Management"
           (DocumentSendingProfile."E-Mail Format" = 'OIOUBL') OR
           (DocumentSendingProfile."Disk Format" = 'OIOUBL'));
     end;
+
```

Comment:
Use the existing OIOUBLFileEvents.FileCreated, no need to create a new event.

Reply:
Yes, that could be a solution, but I wan't to be able to send the file directly from the event and only from the event.
By using the OIOUBLFileEvents.FileCreated, the user would still be asked to download the file or the file would be stored on the disk.  

Fx. if I were to create an extension which send the file directly to a service, the user doesn't want the file exported to neither the server nor download the file, since it has already been sent by the system.

Reply:
OK but then refacture the event with a IsHandled

Reply:
Will do